README.md

```diff
@@ -1,75 +1,19 @@
-    │   │
-    │   ├── data                                <- Scripts to download or generate data
-    │   │   └── make_dataset.py
-    │   │
-    │   ├── features                            <- Scripts to turn raw data into features for modeling
-    │   │   └── build_features.py
-    │   │
-    │   ├── models                              <- Scripts to train models and then use trained models to make
-    │   │   │                                       predictions
-    │   │   ├── predict_model.py
-    │   │   └── train_model.py
-    │   │
-    │   └── visualization                       <- Scripts to create exploratory and results oriented visualizations
-    │       └── visualize.py
-    │
-    ├── .thoth.yaml                             <- Thoth's configuration file
-    ├── .aicoe-ci.yaml                          <- AICoE CI configuration file (https://github.com/AICoE/aicoe-ci)
-    │
-    ├── .github                                 <- GitHub configuration folder
-    │   ├── PULL_REQUEST_TEMPLATE               <- GitHub templates for pull requests
-    │   │
-    │   ├── ISSUE_TEMPLATE                      <- GitHub templates for issues
-    |       ├── {major|minor|patch}_release.md  <- If Khebut GitHub App Bot (https://github.com/apps/khebhut) is installed, the issue will trigger a major|minor|patch release.
-    │       |                                       The bot will open a Pull Request to update the CHANGELOG and fix the opened issue.
-    │       |                                       NOTE: only users that are allowed to release (a.k.a. maintainers specified in the .thoth.yaml file) should open the issue, otherwise bot will
-    │       |                                       reject them, commenting and closing the issue.
-    │       |                                       If AICoE CI GitHub App (https://github.com/apps/aicoe-ci) is installed, once the pull request is merged a new tag is created by the bot
-    │       |                                       and the pipeline to build and push image starts.
-    │       |
-    |       └── redeliver_container_image.md    <- If the tag exists and AICoE CI GitHub App (https://github.com/apps/aicoe-ci) is installed, the issue will retrigger the pipeline to build and
-    │                                               push image container image. NOTE: It should be used if the pipeline triggered with the {major|minor|patch}_release failed for any reason.
-    |
-    └── tox.ini                                 <- tox file with settings for running tox; see tox.readthedocs.io
+We fetch the CI test data from github and further store this to s3 storage. The main requirement here is to get test duration values. We can visualize the distribution of the test_duration metric across various testgrid dashboards and jobs. Based on the distribution type identified, we can find a point after which the test has a higher probability of failing.
```

Comment:
Is testgrid relevant for this project? I thought we were ignoring that and working directly with the github data.

Reply:
I fixed it. My bad.